test/e2e/gateway_single_spoke_test.go

```diff
@@ -293,17 +294,13 @@ var _ = Describe("Gateway single target cluster", func() {
 					By("ensuring the authoritative nameserver resolves the hostname")
 
 					// speed up things by using the authoritative nameserver
-					nameservers, err := net.LookupNS(tconfig.ManagedZone())
+					hname := strings.Replace(tconfig.ManagedZone(), "-", ".", -1)
+
+					nameservers, err := net.LookupNS(hname)
 					Expect(err).ToNot(HaveOccurred())
 					GinkgoWriter.Printf("[debug] authoritative nameserver used for DNS record resolution: %s\n", nameservers[0].Host)
 
-					authoritativeResolver := &net.Resolver{
```

Comment:
@roivaz why did we create our own resolver ? our custom one causes gcp to fail but the default works for both was and gcp

Reply:
@R-Lawton It will be updated quicker on the authoritative ns, which is why it was being used here. This should work for gcp as well btw, and was locally when i was trying it. What's the issue you were seeing?